packages/fxa-settings/src/models/contexts/AppContext.ts

```diff
@@ -28,15 +28,15 @@ export interface AppContextValue {
   account?: Account;
   session?: Session;
   uniqueUserId?: string; // used for experiments
-  experiments?: any; // TODO: add types for experiments
+  experiments?: Promise<any>; // TODO: add types for experiments
```

Comment:
I don't really want to spend the effort of typing this response since it's this external response that feels like it should come with it's own published types. Since we're mapping it out into `NimbusResult` in our hook, I think that is worth more time.